Makefile.am

```diff
@@ -19,7 +19,7 @@ test:
 ########
 
 lib_LTLIBRARIES = src/libsamplerate.la
-include_HEADERS = src/samplerate.h
+include_HEADERS = include/samplerate.h
```

Comment:
What???? Its fine (desirable even) to keep the header file in `src/` and during installation put it in `/usr/include`.

Reply:
I disagree here. Could you tell me your reasoning why it would be "even desirable" to keep the public header in `src/`?

What proved advantageous was to separate out public and private headers already in the source tree. Public headers (ones installed) go into `include` private ones into `src`. This has 2 advantages:
- Tests and examples can be build against the source tree. See https://github.com/erikd/libsamplerate/pull/69/files#diff-af3b638bc2a3e6c650974192a53c7291R70 The binaries linked against libsamplerate don't get to include the private headers. This avoids accidentally using a private there which would break on "real installs" and hence is an additional safe guard.
- Consumers can add this git repo as a submodule and simply use it with `add_subdirectory` (motivation for this PR) They get the same includes (only public ones) as if using the installed library. This avoids having to install this in a separate step (and with a possibly mismatching compiler setting) which is a big bonus especially for Windows where dependencies are just a nightmare.